src/JsonSchema/TypeFactory.php

```diff
@@ -139,7 +139,7 @@ private function getClassType(?string $className, bool $nullable, string $format
         if (true !== $readableLink && $this->isResourceClass($className)) {
             return [
                 'type' => 'string',
-                'format' => 'iri-reference',
+                'format' => 'iri-template',
```

Comment:
I've a problem with that it's not in the specification: https://json-schema.org/understanding-json-schema/reference/string#resource-identifiers isn't redoc wrong?